src/app/calendar/parcelCalendarFunctions.ts

```diff
@@ -15,11 +15,9 @@ export const getParcelsWithCollectionDate = async (): Promise<ParcelWithClientNa
         .from("parcels")
         .select("*, clients ( full_name )")
         .not("collection_datetime", "is", null);
-
     if (error) {
-        throw new Error("Database error");
+        throw new Error("We were unable to fetch the parcels with the specified collection date.");
```

Comment:
Wait just realise it is wrong like the getParcelsWithCollectionDate only filtered those that have null collection_date (there is no specified date if it makes sense)

Reply:
Also do you want to standardize to have "... at this time. Please try again later."